pkg/cvo/availableupdates_test.go

```diff
@@ -0,0 +1,306 @@
+package cvo
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-version-operator/pkg/clusterconditions"
+)
+
+type queueStub struct{}
+
+func (q queueStub) Add(item interface{}) {}
```

Comment:
Nit^2: **Um actually**, we can remove the parameter's name here and on the line 55 as well :nerd_face: 
```suggestion
func (q queueStub) Add(interface{}) {}
```

Reply:
oops, missed that one (and one more)
fixed